contracts/HATVaultsRegistry.sol

```diff
@@ -121,7 +129,11 @@ contract HATVaultsRegistry is Ownable {
     // asset => amount
     mapping(address => uint256) public governanceHatReward;
 
-    HATBountySplit public hatBountySplit;
+    HATBountySplit public defaultHATBountySplit;
+
+    address public defaultArbitrator;
```

Comment:
afaic these vars can also be just alled `arbitrator`, `challengePeriod`, etc.

Reply:
You don't think it's clearer/ more accurate like this?

Reply:
it can also be more confusing in some cases. As in "so this is the defaultArbitrator, where is the actual arbitrator defined?"

Reply:
it is more like a fallback value

Reply:
"default" is a bit ambiguous, which is why we had such different interpretations of how this should be implemented

Reply:
I disagree but not very important tbh. So how you call it? Also rename the event and function name?

Reply:
let's just leave this for now and merge as is